modules/enhancer.js

```diff
@@ -48,30 +48,15 @@ var enhanceWithRadium = function (ComposedComponent: constructor): constructor {
   // with IE <10 any static properties of the superclass aren't inherited and
   // so need to be manually populated
   // See http://babeljs.io/docs/advanced/caveats/#classes-10-and-below-
-  var staticKeys = [
-    'defaultProps',
-    'propTypes',
-    'contextTypes',
-    'childContextTypes'
-  ];
-
-  staticKeys.forEach((key) => {
-    if (ComposedComponent.hasOwnProperty(key)) {
-      RadiumEnhancer[key] = ComposedComponent[key];
+  // This also fixes React Hot Loader by exposing the original components top level
+  // prototype methods on the Radium enhanced prototype as discussed in #219.
+  Object.keys(ComposedComponent.prototype).forEach(key => {
```

Comment:
Thanks for pointing that out @Zenwolf - I completely missed that! `Object.getOwnPropertyNames` is spot on @ianobermiller! :)